### ISSUE TITLE
Create a page to show off past projects

### ISSUE THREAD

**Issue by moorepants** [moorepants/eme185#117]
- http://www.instructables.com/id/DIY-Electric-Powered-Wheelchair/

**Comment by clayfanatic** [comment]
I was looking at some other schools. If we want a page for each project, it would probably be easiest to have the students make the pages, possibly using a template. If we want a list for each year, we could probably link to their posters. I couldn't find the posters, but I did find last year's reports on the google drive.

https://beaverworks.ll.mit.edu/CMS/bw/capstoneprojects
http://me.columbia.edu/SeniorDesign
http://mime.oregonstate.edu/examples-mime-capstone-projects

**Comment by moorepants** [comment]
I agree, having the students make a page on a website would be ideal. I already have plenty of assignments for the class so we'd need to think about what to drop. I also have the video idea too #46. A page with a video would be sweet. I'd prefer it to a poster, in fact.

This also gives the students a tangible digital project that they can always share on job interviews.

One other issue: what if client doesnt' want things shared publicly.

Right now I'm doing these myself manually for solicitation purposes:

http://www.moorepants.info/mech-cap/

**Comment by moorepants** [comment]
I really like the columbia page you shared. Super nice!

**Comment by clayfanatic** [comment]
Where should the pages be hosted? Do we provide students access to a sub-directory on this site? Your .info/mech-cap/ site? Or on a UCD website?

We could have them make their own entirely, but we could possibly loose the site in the future? Is it possible to copy an entire git-hub site? If so, we could have them make a site on git hub, and at the end of their project make a copy of the site.
